<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>697</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>697</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="Container">
     <property name="styleSheet">
      <string notr="true">QFrame#Container{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="Display1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#Display1{
	background-color: #fff;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel#label_2{
	border:5px solid #d9d9d9;
	font: 30px &quot;Oswald&quot;
}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Display2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#Display2{
	background-color:#fff;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel#label{
	border:5px solid #d9d9d9;
	font: 20px &quot;Oswald&quot;
}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Keys">
        <property name="styleSheet">
         <string notr="true">QFrame#Keys{
	background-color: #fff;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="key_container1">
           <property name="styleSheet">
            <string notr="true">QFrame#key_container1{
	background-color:#fff;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_ac">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_ac{
	background-color: #c9302c;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>AC</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_percentage">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_percentage{
	background-color: #286090;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>⁒</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_div">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_div{
	background-color: #286090;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>÷</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_7{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_8{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>8</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_9">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_9{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>9</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_mul">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_mul{
	background-color: #286090;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>×</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_4{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_5{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton_6">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_6{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_sub">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_sub{
	background-color: #286090;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>−</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_1">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_1{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_2{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="pushButton_3">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_3{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="pushButton_add">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_add{
	background-color: #286090;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_ce">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_ce{
	background-color: #c9302c;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>CE</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="key_container2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame#key_container2{
	background-color:#fff;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_0">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_0{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_dot">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_dot{
	background-color: #fff;
	color:#000;
	border:1px solid #000;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>•</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_equl">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_equl{
	background-color: #31b0d5;
	color:#fff;
	font-size:25px;
	border-radius:20px;
	font-weight:600;
}</string>
              </property>
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
